R/monitor.R

```diff
@@ -152,7 +188,13 @@ vetiver_pin_metrics <- function(board,
         vctrs::vec_order(new_metrics[[.index]])
     )
 
-    pins::pin_write(board, new_metrics, basename(metrics_pin_name))
+    pins::pin_write(
+        board,
+        new_metrics,
+        basename(metrics_pin_name),
+        type = old_type,
```

Comment:
vetiver-python has it use the `old_type` only if the user doesn't specify e.g. `type="arrow"`. I wonder if that could be useful, if users want to switch the type?

Reply:
Sure, that sounds good. Added in 66af701b81f595c94835d3b7b91363e45ad7e4ad